README.md

```diff
@@ -35,54 +35,82 @@ cp ./build/bin/bootnode ../bin/bootnode
 bash +x ./setup_bsc_relayer.sh install_k8s
 ```
+5.2. native deploy without docker
+```bash
+rm -rf .local
+
+bash +x ./setup_bc_node.sh native_init // support only one node
+bash +x ./setup_bc_node.sh native_start // can re-entry
+
+bash +x ./setup_bsc_node.sh native_init
+bash +x ./setup_bsc_node.sh native_start // can re-entry
+
+bash +x ./setup_bsc_relayer.sh native_init
+bash +x ./setup_bsc_relayer.sh native_start // can re-entry
+
+bash +x ./setup_oracle_relayer.sh native_init
+bash +x ./setup_oracle_relayer.sh native_start // can re-entry
+```
 
-7. Execute cross chain transaction by sending BNB from BC to BSC
+6. Execute cross chain transaction by sending BNB from BC to BSC
 ```bash
+## 0x9fB29AAc15b9A4B7F17c3385939b007540f4d791 is the address used by test-crosschain-transfer as sender
 ## macos
-echo "12345678" | ./bin/bnbcli bridge transfer-out --amount 50000:BNB --expire-time $(date -v+300S +%s) --to 0x96D904C0e47e6477C4416369a9858f6E57B317eC  --from node0-delegator --chain-id Binance-Chain-Nile --node localhost:26657 --home ./.local/bc/node0
+echo "12345678" | ./bin/tbnbcli bridge transfer-out --amount 500000000:BNB --expire-time $(date -v+300S +%s) --to 0x9fB29AAc15b9A4B7F17c3385939b007540f4d791  --from node0-delegator --chain-id Binance-Chain-Nile --node localhost:26657 --home ./.local/bc/node0
 
 ## linux
-echo "12345678" | ./bin/bnbcli bridge transfer-out --amount 50000:BNB --expire-time $(date --date="+300 seconds" +%s) --to 0x96D904C0e47e6477C4416369a9858f6E57B317eC  --from local-user --chain-id Binance-Chain-Nile --node localhost:26657
+echo "12345678" | ./bin/tbnbcli bridge transfer-out --amount 500000000:BNB --expire-time $(date --date="+300 seconds" +%s) --to 0x9fB29AAc15b9A4B7F17c3385939b007540f4d791  --from local-user --chain-id Binance-Chain-Nile --node localhost:26657
 ```
 
-8. Enable port forwarding
+7. Check the account balance
 ```
-kubectl -n bsc port-forward svc/bsc-node-0 8545:8545
+curl -X POST "http://localhost:8545" -H "Content-Type: application/json"  --data '{"jsonrpc":"2.0","method":"eth_getBalance","params":["0x9fB29AAc15b9A4B7F17c3385939b007540f4d791", "latest"],"id":1}' 
 ```
 
-9. Check the account balance
+8. Execute cross chain transaction by sending BNB from BSC to BC
 ```
-curl -X POST "http://127.0.0.1:8545" -H "Content-Type: application/json"  --data '{"jsonrpc":"2.0","method":"eth_getBalance","params":["0x96D904C0e47e6477C4416369a9858f6E57B317eC", "latest"],"id":1}' 
+
+ ./bin/test-crosschain-transfer --amount 1 --rpc-url http://localhost:8545 --to 0x0cdce3d8d17c0553270064cee95c73f17534d5a0
+```
+
+9. Check the account balance
+``` bash
+## The bech32 format of 0x0cdce3d8d17c0553270064cee95c73f17534d5a0 is tbnb1pnww8kx30sz4xfcqvn8wjhrn796nf4dqshlvlg.(you can use this tool https://slowli.github.io/bech32-buffer/ to do the convert)  
+./bin/tbnbcli account tbnb1pnww8kx30sz4xfcqvn8wjhrn796nf4dqshlvlg --chain-id Binance-Chain-Nile --node localhost:26657 --trust-node  | python3 -m json.tool
```

Comment:
please mark `python3` in the README.md `## Installation`

Reply:
accept and solved